src/Twig/Extensions.php

```diff
@@ -141,9 +142,35 @@ public function getFilters()
                 'orTemplateIfNotExists',
                 [$this, 'doesTwigFileExist']
             ),
+            new \Twig\TwigFilter(
+                'xmlStringCleaner',
+                [$this, 'makeXml']
+            ),
         );
     }
 
+    /**
+     * Return a cleaned-up string representation of a possibly messy string representation of XML input.
+     *
+     * @param string Text representation of XML data, possibly ugly.
+     * @return string A tidyfied cleaned-up string representation of the XML.
```

Comment:
Needs spacing between these two, but return it not needed, unless you want to describe it.